docker/Dockerfile

```diff
@@ -64,6 +64,7 @@ ADD runit/service /var/lib/runit/service
 ADD runit/init.sh /sbin/init.sh
 ADD nginx /etc/nginx
 
+
```

Comment:
Remove newline.

Reply:
@mpereira I've merged the health check related changes from @akirillov https://github.com/mesosphere/spark-build/pull/437/files into my branch. Will fix the whitespace changes.